python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -617,9 +754,9 @@ class BlockingKeyInput(BlockingInput):
     Callable class to retrieve a single keyboard click
     """
     def __init__(self, fig):
-        r"""Create a BlockingKeyInput
+        """Create a BlockingKeyInput
 
-        \param fig The figure to monitor for keyboard events
+        @param fig The figure to monitor for keyboard events
```

Comment:
Maybe switch to numpydoc like some of the changes earlier?

Reply:
This is an orthogonal change that I'd rather do when I'm not rushing to get this into a weekly